Pipfile

```diff
@@ -46,11 +46,13 @@ Flask-Compress = "==1.4.0"
 Flask-Migrate = "==2.1.1"
 Flask-SQLAlchemy = "==2.1"
 Markdown = "==2.6.11"
-PyHive = "==0.5.1"
+PyHive = "==0.6.0"
 SQLAlchemy = "==1.2.2"
 SQLAlchemy-Utils = "==0.32.21"
 thrift_sasl = "==0.3.0"
+sasl = "*"
 pipenv = "*"
+cython = "*"
```

Comment:
We will need this change in setup.py as well right?

Reply:
removed as it was not required